formidable/validators.py

```diff
@@ -194,17 +194,19 @@ def compare(self, value, age):
 
 class ValidatorFactory(object):
 
-    maps = {
-        'MINLENGTH': lambda self, x, y=None: self.min_length(x, y),
-        'MAXLENGTH': lambda self, x, y=None: self.max_length(x, y),
-        'REGEXP': lambda self, x, y=None: self.regexp(x, y),
-        'GT': lambda self, x, y=None: self.gt(x, y),
-        'GTE': lambda self, x, y=None: self.gte(x, y),
-        'LT': lambda self, x, y=None: self.lt(x, y),
-        'LTE': lambda self, x, y=None: self.lte(x, y),
-        'EQ': lambda self, x, y=None: self.eq(x, y),
-        'NEQ': lambda self, x, y=None: self.neq(x, y),
-    }
+    @property
```

Comment:
With @mgu, we were wondering whether it would be nice to use a `cached_property` here, but it looks like this method is not actually called several times with the same instance.

Reply:
@wo0dyn I thought about it. Just to be sure, do you mean this lib https://pypi.python.org/pypi/cached-property or a simple None verification for the calls?

Reply:
Django has its own `cached_property` from [`django.utils.functional`](https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.functional.cached_property). 😉 
Maybe it doesn't hurt that much to use it here anyway.

Reply:
Thank you for information, I never used it. I think that better to add it because it will allow avoiding recreation of the dictionary on each field validation (And we make a call to this property in a loop).